python/lib/dicom/dicom_database.py

```diff
@@ -0,0 +1,231 @@
+from datetime import datetime
```

Comment:
This file should go under the `database_lib` directory. This is where we store the DB interaction files. The files are organized by table names and holds the table specific queries. It would be best to keep that organization.